pocWebsite.html

```diff
@@ -0,0 +1,381 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Bootstrap 5 Example</title>
+    <meta charset="utf-8">
+    <!-- Latest compiled and minified CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <!-- Latest compiled JavaScript -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <script>
+        function openTab(evt, cardName) {
+            // Declare all variables
+            let i, cardcontent, cardlinks
+
+            // Get all elements with class="cardcontent" and hide them
+            cardcontent = document.getElementsByClassName("cardcontent")
+            for (i = 0; i < cardcontent.length; i++) {
+                cardcontent[i].style.display = "none"
+            }
+
+            // Get all elements with class="cardlinks" and remove the class "active"
+            cardlinks = document.getElementsByClassName("cardlinks")
+            for (i = 0; i < cardlinks.length; i++) {
+                cardlinks[i].className = cardlinks[i].className.replace(" active", "")
+            }
+
+            // Show the current card, and add an "active" class to the button that opened the card
+            document.getElementById(cardName).style.display = "block"
+            evt.currentTarget.className += " active"
+        }
+        document.addEventListener('DOMContentLoaded', function () {
+            document.getElementById("autobtn").click()
+        }, false)
+
+    </script>
+    <style>
+        .wrapper {
+            position: relative;
+        }
+
+        .cardColumn {
+            width: 25%;
+            align-items: stretch;
+        }
+
+        .card {
+            height: 100%;
+            text-align: center;
+        }
+
+        #auto,
+        #home,
+        #commercial,
+        #life {
+            height: 100%;
+            background-color: #3e87f5;
+        }
+
+        .card-img-top {
+            align-items: center;
+        }
+
+        .autobtn,
+        .homebtn,
+        .buisbtn,
+        .lifebtn {
+            position: absolute;
+            text-align: center;
+        }
+    </style>
+</head>
+
+<body>
+</body>
+
+</html>
+<div class="container-fluid p-5 bg-primary text-white">
```

Comment:
all of this should be in the body

Reply:
also as a note for both myself and you, we should probably be putting an image with visual changes like this as well, after all can't really say too much in terms of the appearance because I don't see how it looks.